orchagent/orchdaemon.cpp

```diff
@@ -760,6 +765,15 @@ void OrchDaemon::start()
             continue;
         }
 
+        // output heart beat message to SYSLOG
```

Comment:
>beat [](http://example.com/codeflow?start=24&length=4)

In the above part of while-loop block, there are several `continue` statements. So there is flow like early loop. Is it possible that early loop like timeout prevent heart beat?

Reply:
Fixed, move code to new method and send message at beginning of every loop.